src/mito2/src/sst/index/bloom_filter/applier/builder.rs

```diff
@@ -178,74 +161,15 @@ impl<'a> BloomFilterIndexApplierBuilder<'a> {
             return Ok(());
         };
         let value = encode_lit(lit, data_type)?;
-
-        // Create bloom filter predicate
-        let mut set = HashSet::new();
-        set.insert(value);
-        let predicate = Predicate::InList(InListPredicate { list: set });
-
-        // Add to output predicates
-        self.output.entry(column_id).or_default().push(predicate);
-
-        Ok(())
-    }
-
-    /// Collects an in list expression in the form of `column IN (lit, lit, ...)`.
-    fn collect_in_list(&mut self, in_list: &InList) -> Result<()> {
-        // Only collect InList predicates if they reference a column
-        let Expr::Column(column) = &in_list.expr.as_ref() else {
-            return Ok(());
-        };
-        if in_list.list.is_empty() || in_list.negated {
-            return Ok(());
-        }
-
-        let Some((column_id, data_type)) = self.column_id_and_type(&column.name)? else {
-            return Ok(());
-        };
-
-        // Convert all non-null literals to predicates
-        let predicates = in_list
-            .list
-            .iter()
-            .filter_map(Self::nonnull_lit)
-            .map(|lit| encode_lit(lit, data_type.clone()));
-
-        // Collect successful conversions
-        let mut valid_predicates = HashSet::new();
-        for predicate in predicates {
-            match predicate {
-                Ok(p) => {
-                    valid_predicates.insert(p);
-                }
-                Err(e) => warn!(e; "Failed to convert value in InList"),
-            }
-        }
-
-        if !valid_predicates.is_empty() {
-            self.output
-                .entry(column_id)
-                .or_default()
-                .push(Predicate::InList(InListPredicate {
-                    list: valid_predicates,
-                }));
-        }
+        self.probes.entry(column_id).or_default().insert(value);
```

Comment:
So the skipping index doesn't support `IN` now?

Reply:
working on it